### PR TITLE
Update Developing.md

### PR DESCRIPTION
**Pull request by MarkZ3**
Mention that yarn v1.0.2 must be used.

### CODE REVIEW

File: doc/Developing.md [hunk 1]
```diff
@@ -250,6 +254,12 @@ errors while building, testing or running the application then;
  operating systems. Hence we are more than happy to receive any Windows-related
  feedbacks, bug reports.
 
+If you have accidentally installed the wrong yarn version, you have to remove it, then install it.
+
+ - Run PowerShell as an administrator.
+ - Run: choco uninstall yarn -y.
```
Comment (kittaakos) [inline doc/Developing.md]: Please remove the trailing dot, I left it there.

File: doc/Developing.md [hunk 2]
```diff
@@ -250,6 +254,12 @@ errors while building, testing or running the application then;
  operating systems. Hence we are more than happy to receive any Windows-related
  feedbacks, bug reports.
 
+If you have accidentally installed the wrong yarn version, you have to remove it, then install it.
```
Comment (kittaakos) [inline doc/Developing.md]: Please escape `yarn`. Thank you!

Comment (kittaakos) [reply doc/Developing.md]: then install it. => then reinstall it.

Comment (MarkZ3) [reply doc/Developing.md]: ok. Done!

Comment (kittaakos) [reply doc/Developing.md]: Thank you!

**Pull request merged**